src/server/renderer.js

```diff
@@ -44,14 +44,14 @@ export default (projectConfig, projectToolsConfig) => {
           console.error('ROUTER ERROR:', pretty.render(error));
           res.status(500);
         } else if (renderProps) {
-          loadPropsOnServer(renderProps, (err, asyncProps) => {
+          loadOnServer(renderProps, store).then(() => {
```

Comment:
@sompylasar , yes, you are right, thanks.
I fixed this: https://github.com/Rezonans/redux-async-connect/commit/6b2cf89adb90fb732c6c76df65c9f11a7e832548